### ISSUE TITLE
Running the command giving my error for "Cannot find module 'fs/promises'"

### ISSUE THREAD

**Issue by lucky13820** [NyaomiDEV/AMDFriend#2]
```
internal/modules/cjs/loader.js:984
  throw err;
  ^

Error: Cannot find module 'fs/promises'
Require stack:
- /Users/myuser/.nvm/versions/node/v12.16.0/lib/node_modules/amdfriend/dist/index.js
- /Users/myuser/.nvm/versions/node/v12.16.0/lib/node_modules/amdfriend/dist/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:981:15)
    at Function.Module._load (internal/modules/cjs/loader.js:863:27)
    at Module.require (internal/modules/cjs/loader.js:1043:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/Users/ryanyao/.nvm/versions/node/v12.16.0/lib/node_modules/amdfriend/dist/index.js:9:20)
    at Module._compile (internal/modules/cjs/loader.js:1157:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1177:10)
    at Module.load (internal/modules/cjs/loader.js:1001:32)
    at Function.Module._load (internal/modules/cjs/loader.js:900:14)
    at Module.require (internal/modules/cjs/loader.js:1043:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/myuser/.nvm/versions/node/v12.16.0/lib/node_modules/amdfriend/dist/index.js',
    '/Users/myuser/.nvm/versions/node/v12.16.0/lib/node_modules/amdfriend/dist/cli.js'
  ]
}

```

Not sure where I did wrong.

**Comment by lucky13820** [comment]
fixed by using Node 14.3.0

**Comment by NyaomiDEV** [comment]
Yeah, that Node version was prettyyyy old
